core/controllers/editor.py

```diff
@@ -312,10 +325,11 @@ def get(self, exploration_id):
         if not rights_manager.Actor(self.user_id).can_view(
                 rights_manager.ACTIVITY_TYPE_EXPLORATION, exploration_id):
             raise self.PageNotFoundException
-
         version = self.request.get('v', default_value=None)
+        apply_draft = self.request.get('apply_draft', default_value=False)
```

Comment:
Why are you giving the frontend the option to not include draft changes? That seems like it would lead to a lot of problems, as the user will now be autosaving new draft changes that are incompatible with the old ones.